tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts

```diff
@@ -0,0 +1,188 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Although git did not think so, this is effectively a move from tf-audio-dashboard.html and we should keep the license year there. Change this to 2016.

Reply:
Done